api/estados/saopaulo.js

```diff
@@ -1,652 +1,654 @@
+    'Indiana',
+    'Indiaporã',
+    'Inúbia Paulista',
+    'Ipauçu',
+    'Iperó',
+    'Ipeúna',
+    'Ipiguá',
+    'Iporanga',
+    'Ipuã',
+    'Iracemápolis',
+    'Irapuã',
+    'Irapuru',
+    'Itaberá',
+    'Itaí',
+    'Itajobi',
+    'Itaju',
+    'Itanhaém',
+    'Itaóca',
+    'Itapecerica da Serra',
+    'Itapetininga',
+    'Itapeva',
+    'Itapevi',
+    'Itapira',
+    'Itapirapuã Paulista',
+    'Itápolis',
+    'Itaporanga',
+    'Itapuí',
+    'Itapura',
+    'Itaquaquecetuba',
+    'Itararé',
+    'Itariri',
+    'Itatiba',
+    'Itatinga',
+    'Itirapina',
+    'Itirapuã',
+    'Itobi',
+    'Itu',
+    'Itupeva',
+    'Ituverava',
+    'Jaborandi',
+    'Jaboticabal',
+    'Jacareí',
+    'Jaci',
+    'Jacupiranga',
+    'Jaguariúna',
+    'Jales',
+    'Jambeiro',
+    'Jandira',
+    'Jardinópolis',
+    'Jarinu',
+    'Jaú',
+    'Jeriquara',
+    'Joanópolis',
+    'João Ramalho',
+    'José Bonifácio',
+    'Júlio Mesquita',
+    'Jumirim',
+    'Jundiaí',
+    'Junqueirópolis',
+    'Juquiá',
+    'Juquitiba',
+    'Lagoinha',
+    'Laranjal Paulista',
+    'Lavínia',
+    'Lavrinhas',
+    'Leme',
+    'Lençóis Paulista',
+    'Limeira',
+    'Lindóia',
+    'Lins',
+    'Lorena',
+    'Lourdes',
+    'Louveira',
+    'Lucélia',
+    'Lucianópolis',
+    'Luís Antônio',
+    'Luiziânia',
+    'Lupércio',
+    'Lutécia',
+    'Macatuba',
+    'Macaubal',
+    'Macedônia',
+    'Magda',
+    'Mairinque',
+    'Mairiporã',
+    'Manduri',
+    'Marabá Paulista',
+    'Maracaí',
+    'Marapoama',
+    'Mariápolis',
+    'Marília',
+    'Marinópolis',
+    'Martinópolis',
+    'Matão',
+    'Mauá',
+    'Mendonça',
+    'Meridiano',
+    'Mesópolis',
+    'Miguelópolis',
+    'Mineiros do Tietê',
+    'Mira Estrela',
+    'Miracatu',
+    'Mirandópolis',
+    'Mirante do Paranapanema',
+    'Mirassol',
+    'Mirassolândia',
+    'Mococa',
+    'Mogi das Cruzes',
+    'Mogi-Guaçu',
+    'Mogi-Mirim',
+    'Mombuca',
+    'Monções',
+    'Mongaguá',
+    'Monte Alegre do Sul',
+    'Monte Alto',
+    'Monte Aprazível',
+    'Monte Azul Paulista',
+    'Monte Castelo',
+    'Monte Mor',
+    'Monteiro Lobato',
+    'Morro Agudo',
+    'Morungaba',
+    'Motuca',
+    'Murutinga do Sul',
+    'Nantes',
+    'Narandiba',
+    'Natividade da Serra',
+    'Nazaré Paulista',
+    'Neves Paulista',
+    'Nhandeara',
+    'Nipoã',
+    'Nova Aliança',
+    'Nova Campina',
+    'Nova Canaã Paulista',
+    'Nova Castilho',
+    'Nova Europa',
+    'Nova Granada',
+    'Nova Guataporanga',
+    'Nova Independência',
+    'Nova Luzitânia',
+    'Nova Odessa',
+    'Novais',
+    'Novo Horizonte',
+    'Nuporanga',
+    'Ocauçu',
+    'Óleo',
+    'Olímpia',
+    'Onda Verde',
+    'Oriente',
+    'Orindiúva',
+    'Orlândia',
+    'Osasco',
+    'Oscar Bressane',
+    'Osvaldo Cruz',
+    'Ourinhos',
+    'Ouro Verde',
+    'Ouroeste',
+    'Pacaembu',
+    'Palestina',
+    'Palmares Paulista',
+    "Palmeira d'Oeste",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)